app/assets/stylesheets/components/_review-box.scss

```diff
@@ -25,4 +25,14 @@
       line-height: 3rem;
     }
   }
-}
\ No newline at end of file
+
+  &__button {
+    border: 2px solid black;
+    line-height: 2px;
+  }
+}
```

Comment:
We might check with Anu to see if this is a new consistent button type, and refactor it into the buttons css file